mantidproject/mantid
# Fix Qt warnings around COMPAT signals

After removing Qt3Support there were warnings remaining when starting MantidPlot using a debug build under Visual Studio. The signals have been updated to use the ones still present in Qt 4 & 5.

Note that the warnings are only present when using a debug build of **Qt**, a debug build of mantid on Linux is not enough, so this is most easily tested using a debug build on Windows.  

**To test:**

Build the code and start MantidPlot. There should be no warnings regarding using a `COMPAT` signal.

Fixes #16586 

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
